### PR TITLE
hello-micrroverse

### PR DESCRIPTION
**Pull request by ausiemass**
The README.md file has been edited.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,6 @@
-# hello-microverse
\ No newline at end of file
+# hello-microverse
+### Hello microverse is my first project at Microverse. It is a basic project to explain how to create branches and Pull requests.
+
+## Built With
+### the project is built with Html and CSS as well as javascipt
+
```
Comment (roshan-bajgain) [inline README.md]: - [ ] I would suggest you please add a descriptive README file to your project. please use [this template](https://github.com/microverseinc/readme-template) and make sure that every heading should be present on your README file, the only optional is FAQ. And If you decide not to deploy the app please remove the live demo. Please write 1,2 sentences about all heading in your README file.

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @ausiemass, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
- [ ] I would suggest you please write the descriptive title and pull request telling you what changes are inside. like: You can use `Hello Microverse` title for your pull request.
`Add Descriptive README file`
`Setup linter correctly`
`Add index and css file`
`Add node_module and test.md in .gitignore file` etc.
- [ ] I would suggest you please use the branch name of a feature or action like you can rename the branch name from first branch to `hello-microverse`.
- [ ] All the linter should be pass and Linter result should be green it helps to write clean code and follow best practice. To fix this error run `npx stylelint "**/*.{css,scss}"` in you terminal and it will show error and resolve it.
### Error
Screenshot 2023-06-02 071643

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TEMESGEN-G25 — CHANGES_REQUESTED** [review]
## :yellow_circle: CHANGES REQUIRED :yellow_circle: 

## Hi @ausiemass :wave:, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Highlights
:heavy_check_mark: Good commits.

 ## Required Changes ♻️
- [x]  Please, try to match the [README template](https://github.com/microverseinc/readme-template) Microverse has provided. 
- The 'MIT' link in the License section isn't working. 
- The links in the Authors' section aren't working along with having two different authors, where neither of them are obviously you.
- The headers are missing.
- The 'back to top' links aren't working.

- [ ]  I recommend changing your pull request title & description to a more meaningful one. Currently, it's not giving any context about your project. This is important because other developers or potential hiring recruiters are able to understand & know how your project works. Although this is a 'Hello Microverse' project, our goal is to make this a habit for you.

- [ ] I suggest that you rename your branch into a more meaningful one. Like `hello-microverse`. Having a more meaningful branch name helps in reducing complexity when working on huge projects. Although this isn't a huge project, our goal is to make this a habit.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TEMESGEN-G25 — APPROVED** [review]
## :green_circle:  APPROVED :green_circle: 

## Hi @ausiemass :wave: :smile:, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
:heavy_check_mark: Great README.
:heavy_check_mark: Awesome PR title & description.



## Optional suggestions

- [ ] At the pull request (PR), I suggest adding a short summary of your changes. Like what you did, what tech stack you used, or how you implemented your changes. This will help your PR become easily understandable & help other reviewers leave clearer & better suggestions.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**